bundles/com.espressif.idf.debug.gdbjtag.openocd/src/com/espressif/idf/debug/gdbjtag/openocd/dsf/console/IdfProcessConsoleSettingsHandler.java

```diff
@@ -0,0 +1,24 @@
+/*******************************************************************************
+ * Copyright 2023 Espressif Systems (Shanghai) PTE LTD. All rights reserved.
+ * Use is subject to license terms.
+ *******************************************************************************/
+package com.espressif.idf.debug.gdbjtag.openocd.dsf.console;
+
+import org.eclipse.core.commands.AbstractHandler;
+import org.eclipse.core.commands.ExecutionEvent;
+import org.eclipse.core.commands.ExecutionException;
+
+/**
+ * The command handler to open the settings page for the process console.
+ * @author Ali Azam Rana
+ *
+ */
+public class IdfProcessConsoleSettingsHandler extends AbstractHandler
+{
+	@Override
+	public Object execute(ExecutionEvent event) throws ExecutionException
+	{
+		// TODO This will open the preferences page for the console settings
+		return null;
+	}
+}
```

Comment:
Couldn't find any references for this, can we clean this up?

Reply:
done